### ISSUE TITLE
[Bug]: Multiple sources type invalid config error

### ISSUE THREAD

**Issue by Chipsu24** [mampfes/hacs_waste_collection_schedule#1867]
### I Have A Problem With:

A specific source

### What's Your Problem

After the integration update I tried to modify my existing configuration to be only one calendar entity with multiple waste types. Home Assistant notifies me of a configuration error. I double checked the examples for this type and found no errors in my config

### Source (if relevant)

multiple

### Logs

```Shell
Logger: homeassistant.config
Source: config.py:1293
First occurred: 21:46:30 (1 occurrences)
Last logged: 21:46:30

Invalid config for 'waste_collection_schedule' at waste_collection.yaml, line 3: expected dict for dictionary value 'waste_collection_schedule->sources->0->args', got [{'static': [{'type': 'Zmieszane', 'frequency': 'WEEKLY', 'interval': 2, 'start': '2024-01-09', 'until': '2024-12-24', 'customize': {'type': 'Zmieszane', 'icon': 'phu:garbage-residual', 'calendar_title': 'Odpady Zmieszane'}}]}, {'static': [{'type': 'Segregowane', 'dates': ['2024-01-19', '2024-02-16', '2024-03-18', '2024-04-18', '2024-05-20', '2024-06-18', '2024-07-17', '2024-08-20', '2024-09-18', '2024-10-16', '2024-11-20', '2024-12-17'], 'customize': {'type': 'Segregowane', 'icon': 'mdi:recy..., please check the docs at https://github.com/mampfes/hacs_waste_collection_schedule#readme
```

### Relevant Configuration

```YAML
#the waste_collection_schedule: is in configuration.yaml with this config !included (worked before changing)  
sources:
    - name: multiple
      args:
        - static:
          - type: "Zmieszane"
            frequency: WEEKLY
            interval: 2
            start: '2024-01-09'
            until: '2024-12-24'
            customize:
              type: Zmieszane
              icon: phu:garbage-residual
              calendar_title: "Odpady Zmieszane"
        - static:
          - type: Segregowane
            dates:
              - '2024-01-19'
              - '2024-02-16'
              - '2024-03-18'
              - '2024-04-18'
              - '2024-05-20'
              - '2024-06-18'
              - '2024-07-17'
              - '2024-08-20'
              - '2024-09-18'
              - '2024-10-16'
              - '2024-11-20'
              - '2024-12-17'
            customize:
              type: Segregowane
              icon: mdi:recycle-variant
              calendar_title: "Odpady Segregowane"
        - static:
          - type: Bio
            dates:
              - '2024-01-25'
              - '2024-02-22'
              - '2024-03-21'
              - '2024-04-11'
              - '2024-04-25'
              - '2024-05-09'
              - '2024-05-23'
              - '2024-06-06'
              - '2024-06-20'
              - '2024-07-04'
              - '2024-07-18'
              - '2024-08-01'
              - '2024-08-16'
              - '2024-08-29'
              - '2024-09-12'
              - '2024-09-29'
              - '2024-10-10'
              - '2024-10-24'
              - '2024-11-07'
              - '2024-11-21'
              - '2024-12-12'
            customize:
              type: Bio
              icon: mdi:leaf-circle-outline
              calendar_title: "Odpady Bio"
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [ ] Checked that the website of your service provider is still working
- [ ] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [ ] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by anym001** [comment]
I have the same problem. 
Also get an error message with this config.

**Comment by 5ila5** [comment]
Sorry the documentation was wrong, the sources should be a dictionary and not a list, fixed that in #1873
